src/js/containers/covid19/Covid19Container.jsx

```diff
@@ -0,0 +1,111 @@
+/**
+ * Covid19Container.jsx
+ * Created by Jonathan Hill 06/02/20
+ */
+
+import React, { useState } from 'react';
+import { startCase, snakeCase } from 'lodash';
+import Cookies from 'js-cookie';
+import MetaTags from 'components/sharedComponents/metaTags/MetaTags';
+import Header from 'components/sharedComponents/header/Header';
+import Sidebar from 'components/sharedComponents/sidebar/Sidebar';
+import StickyHeader from 'components/sharedComponents/stickyHeader/StickyHeader';
+import Covid19Section from 'components/covid19/Covid19Section';
+import Footer from 'containers/Footer';
+// import { Picker } from 'data-transparency-ui';
+import ShareIcon from 'components/sharedComponents/stickyHeader/ShareIcon';
+// import { defaultSortFy } from 'components/sharedComponents/pickers/FYPicker';
+import FooterLinkToAdvancedSearchContainer from 'containers/shared/FooterLinkToAdvancedSearchContainer';
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { covidPageMetaTags } from 'helpers/metaTagHelper';
+import { jumpToSection } from 'helpers/covid19Helper';
+import {
+    slug,
+    getEmailSocialShareData,
+    scrollPositionOfSiteHeader,
+    componentByCovid19Section,
+    footerTitle,
+    footerDescription
+} from 'dataMapping/covid19/covid19';
+
+require('pages/covid19/index.scss');
+
+const Covid19Container = () => {
+    const [activeSection, setActiveSection] = useState('overview');
+    // const [selectedDEF, setselectedDEF] = useState('All');
+
+    // const DEFOptions = getDEFOptions(setselectedDEF, defaultSortFy);
+
+    const jumpToCovid19Section = (section) => jumpToSection(section, activeSection, setActiveSection);
+
+    return (
+        <div className="usa-da-covid19-page">
+            <MetaTags {...covidPageMetaTags} />
+            <Header />
+            <StickyHeader>
+                <>
```

Comment:
For some reason this `<` is showing up red for me?

Reply:
Not sure, it is not for me.

Reply:
Yeah, the page renders fine, so I think we're good!